pylint/checkers/misc.py

```diff
@@ -90,18 +90,35 @@ class EncodingChecker(BaseTokenChecker, BaseRawFileChecker):
                 "default": "",
             },
         ),
+        (
+            "check-fixme-in-docstring",
+            {
+                "type": "yn",
+                "metavar": "<y or n>",
+                "default": False,
+                "help": "Whether or not to search for fixme's in docstrings.",
+            },
+        ),
     )
 
     def open(self) -> None:
         super().open()
 
         notes = "|".join(re.escape(note) for note in self.linter.config.notes)
         if self.linter.config.notes_rgx:
-            regex_string = rf"#\s*({notes}|{self.linter.config.notes_rgx})(?=(:|\s|\Z))"
+            comment_regex = (
+                rf"#\s*({notes}|{self.linter.config.notes_rgx})(?=(:|\s|\Z))"
+            )
+            self._comment_fixme_pattern = re.compile(comment_regex, re.I)
+            if self.linter.config.check_fixme_in_docstring:
+                docstring_regex = rf"((\"\"\")|(\'\'\'))\s*({notes}|{self.linter.config.notes_rgx})(?=(:|\s|\Z))"
+                self._docstring_fixme_pattern = re.compile(docstring_regex, re.I)
         else:
-            regex_string = rf"#\s*({notes})(?=(:|\s|\Z))"
-
-        self._fixme_pattern = re.compile(regex_string, re.I)
+            comment_regex = rf"#\s*({notes})(?=(:|\s|\Z))"
+            self._comment_fixme_pattern = re.compile(comment_regex, re.I)
+            if self.linter.config.check_fixme_in_docstring:
+                docstring_regex = rf"((\"\"\")|(\'\'\'))\s*({notes})(?=(:|\s|\Z))"
+                self._docstring_fixme_pattern = re.compile(docstring_regex, re.I)
```

Comment:
I think we need to always set these two pattern attributes so that we don't get an `AttributeError` in unexpected ways. In the old version we also did this.

Reply:
Gotcha, I've removed the conditionals here and just let `process_tokens()` handle the logic of what to parse if check docstring for fixme setting is enabled.